### ISSUE TITLE
How to decrease the performance gap between the val set and test set?

### ISSUE THREAD

**Issue by PkuRainBow** [CSAILVision/semantic-segmentation-pytorch#63]
Hi, I have trained some models and achieved new state-of-the-art performance on the val set of the cityscapes.

But when I submit the results on the test set. I find the performance gap is very large! 

With single crop, the performance on the val set will drop 2 points on the test set. I guess maybe it is related to the DSN structure or related to the learning rate  and so on.

However, my previous methods without DSN only suffers 1 point drop between the val set and test set.

Could you help me on how to avoid this gap?
